lessons/Early-Math/computation/langs/en-us/index.adoc

```diff
@@ -0,0 +1,236 @@
+= Circles, Computation, and Equivalence
+
+@description{Students explore _computation_ as a way to generate equivalent expressions and Circles.}
+
+@lesson-prereqs{translating}
+
+@keywords{computation, equivalence}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Use computation to develop equivalent representations of a given expression.
+* Diagram solving with Circles using Computation.
```

Comment:
I am not able to parse what these goals are intended to mean.

Reply:
This lesson plan often uses "Circles" in place of "Circles of Evaluation". I highly encourage you to use all of the words. I made the change for you in the first lesson and there were only one or two instances in the entire lesson plan where using just the word circle made sense to me and it was because you were referring to the shape itself rather than the shape and the numbers and the operator.

Reply:
In general, please use Circles of Evaluation instead of Circles almost all of the time. I have now updated two entire lesson plans and have found only a handful of spots where the word circle made sense in lieu of Circles of Evaluation.